src/mpid/ch4/shm/xpmem/xpmem_send.h

```diff
@@ -43,24 +41,8 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_XPMEM_lmt_isend(const void *buf, MPI_Aint cou
 
     MPIR_Assert(is_contig && data_sz > 0);
 
-    /* XPMEM internal info */
-    slmt_req_hdr->src_offset = (uint64_t) buf + true_lb;
-    slmt_req_hdr->data_sz = data_sz;
-    slmt_req_hdr->sreq_ptr = (uint64_t) sreq;
-    slmt_req_hdr->src_lrank = MPIDI_XPMEM_global.local_rank;
-
-    /* message matching info */
-    slmt_req_hdr->src_rank = comm->rank;
-    slmt_req_hdr->tag = tag;
-    slmt_req_hdr->context_id = comm->context_id + context_offset;
-
-    XPMEM_TRACE("lmt_isend: shm ctrl_id %d, src_offset 0x%lx, data_sz 0x%lx, sreq_ptr 0x%lx, "
-                "src_lrank %d, match info[dest %d, src_rank %d, tag %d, context_id 0x%x]\n",
-                MPIDI_SHM_XPMEM_SEND_LMT_RTS, slmt_req_hdr->src_offset,
-                slmt_req_hdr->data_sz, slmt_req_hdr->sreq_ptr, slmt_req_hdr->src_lrank,
-                rank, slmt_req_hdr->src_rank, slmt_req_hdr->tag, slmt_req_hdr->context_id);
-
-    mpi_errno = MPIDI_SHM_do_ctrl_send(rank, comm, MPIDI_SHM_XPMEM_SEND_LMT_RTS, &ctrl_hdr);
```

Comment:
Is the trace message lost?

Reply:
It does. :) I could add it back, or as I started in another PR, removed all traces. I find these traces are only used during development or debugging and work like scaffold. They can be easily added back on-demand if a debugging session need them and should not be left in production code.